contracts/Comet.sol

```diff
@@ -763,18 +764,52 @@ contract Comet is CometMainInterface {
      * @dev Safe ERC20 transfer in, assumes no fee is charged and amount is transferred
      */
     function doTransferIn(address asset, address from, uint amount) internal {
-        bool success = ERC20(asset).transferFrom(from, address(this), amount);
+        bool success = safeTransfer(asset, from, address(this), amount);
         if (!success) revert TransferInFailed();
     }
 
     /**
      * @dev Safe ERC20 transfer out
      */
     function doTransferOut(address asset, address to, uint amount) internal {
-        bool success = ERC20(asset).transfer(to, amount);
+        bool success = safeTransfer(asset, address(this), to, amount);
         if (!success) revert TransferOutFailed();
     }
 
+    /**
+     * @dev Helper function that properly supports `transfer` and `transferFrom` for non-standard ERC-20 tokens, returning a bool of the transfer's success
+     * if the transfer is successful or not.
+     * @dev Note: This does not check that the amount transferred in is actually equals to the amount specified (e.g. fee tokens will not revert)
+     * @dev Note: This wrapper safely handles non-standard ERC-20 tokens that do not return a value. See here: https://medium.com/coinmonks/missing-return-value-bug-at-least-130-tokens-affected-d67bf08521ca
+     * @param asset The ERC-20 token to transfer
+     * @param from The address to transfer from
+     * @param from The address to transfer from
```

Comment:
this is repeated above

Reply:
oops,  I think this mistake was made during rebasing 😓